Dockerfile.pipelines

```diff
@@ -35,6 +35,12 @@ ENV testWithPip=$installWithPip
 ARG DEVITO_BACKEND=none
 ENV DEVITO_BACKEND=$DEVITO_BACKEND
 
+# Set OPS variable in case they are needed
+ARG ops_compiler=gnu
+ARG ops_install_path=/tmp/OPS/ops
+ENV OPS_COMPILER=$ops_compiler
```

Comment:
probably you can drop this, for the same reasons as above? what's its purpose otherwise?

Reply:
this env var is required by ops` makefile